database/migrations/2020_09_02_132413_update_users_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('profile_img')->nullable()->change();
```

Comment:
I believe id_img is nullable too, so you can change it like profile_img

Reply:
resolved [here](https://github.com/BuildForSDGCohort2/Team-248-Backend/pull/29/commits/81796055bdb7020d0c9d324e6339584117ebca38)